library/igw_purge.py

```diff
@@ -70,7 +70,8 @@ def delete_group(module, image_list, cfg):
 def delete_rbd(module, rbd_path):
 
     logger.debug("issuing delete for {}".format(rbd_path))
-    rm_cmd = 'rbd --no-progress rm {}'.format(rbd_path)
+    rm_cmd = 'rbd --no-progress --conf {} rm {}'.format(settings.config.cephconf,
```

Comment:
The file exists in 2 places so we can distribute the module as part of the ceph-iscsi role in the ansible-galaxy.

Reply:
@leseb I'm not familiar enough w/ Ansible to know what that means, but assuming it's just a place to share code, I guess I still don't understand why we need to copy-and-paste code around.

Reply:
@dillaman basically ceph-ansible as a repo is an all-in-one tree that has all the roles, typically modules are stored at the root in the library directory. When pushing roles to the ansible-galaxy we have 1 repo (e.g https://github.com/ceph/ansible-ceph-common/tree/master/plugins/actions) per role. We use git subtree to create individual git trees for each role and push them to the galaxy.
Since our modules are not packaged in pip that's the only way we can have them in ceph-ansible AND in the galaxy repo. So we duplicate the module. Hope that clarifies a bit.